src/isolation/filemap.rs

```diff
@@ -90,6 +92,11 @@ impl UhyveFileMap {
 		self.files.values().map(|i| i.as_os_str())
 	}
 
+	/// Returns an array of all guest paths.
+	pub(crate) fn get_all_guest_paths(&self) -> impl Iterator<Item = &str> {
+		self.files.keys().map(|i| i.to_str().unwrap())
```

Comment:
If we require all keys to be strings, then the type of the map `files` should be changed instead.

Reply:
https://github.com/rust-vmm/vm-fdt/pull/75 should be fixable.

Reply:
This is a fix for it, but I don't think we should wait for it to be released and just keep the code as is: https://github.com/rust-vmm/vm-fdt/pull/76